src/plugins/Tus10.js

```diff
@@ -96,6 +97,16 @@ module.exports = class Tus10 extends Plugin {
           updatedFiles[file] = updatedFile
         })
         this.core.setState({files: updatedFiles})
+        return
+      case 'retryAll':
+        inProgressUpdatedFiles.forEach((file) => {
+          const updatedFile = Object.assign({}, updatedFiles[file], {
+            isPaused: false,
+            error: null
+          })
+          updatedFiles[file] = updatedFile
+        })
+        this.core.setState({files: updatedFiles})
```

Comment:
Ye, this only touches core. I think all of these `case`s would make more sense to do in core, controlled by events. (`core:pause-all`, `core:resume-all`, `core:retry-all`) Same for `case 'toggle':`, which seems to respond to `core:upload-pause`.

Reply:
Moved to Core.